doc/mgr/orchestrator.rst

```diff
@@ -180,9 +183,25 @@ Example::
     all-available-devices node2 /dev/vdc -  -
     all-available-devices node3 /dev/vdd -  -
 
+When it is used the parameter ``all-available-devices``, a cephadm service is created.
```

Comment:
s/ When it is used the parameter ``all-available-devices``, a cephadm service is created./ When the parameter ``all-available-devices`` is used, a cephadm service is created/